crates/oxc_linter/src/rules/jest/consistent_test_it.rs

```diff
@@ -0,0 +1,916 @@
+        (
+            "describe(\"suite\", () => { test(\"foo\") })",
+            Some(serde_json::json!([{ "withinDescribe": "it" }])),
+        ),
+        // consistent-test-it with withinDescribe=test
+        ("it(\"foo\")", Some(serde_json::json!([{ "withinDescribe": "test" }]))),
+        (
+            "describe(\"suite\", () => { it(\"foo\") })",
+            Some(serde_json::json!([{ "withinDescribe": "test" }])),
+        ),
+    ];
+
+    let mut fix = vec![
+        // consistent-test-it with fn=test
+        ("it(\"foo\")", "test(\"foo\")"),
+        (
+            "
+                import { it } from '@jest/globals';
+                it(\"foo\")
+            ",
+            "
+                import { it } from '@jest/globals';
+                test(\"foo\")
+            ",
+        ),
+        (
+            "
+                import { it as testThisThing } from '@jest/globals';
+                testThisThing(\"foo\")
+            ",
+            "
+                import { it as testThisThing } from '@jest/globals';
+                test(\"foo\")
+            ",
+        ),
+        ("xit(\"foo\")", "xtest(\"foo\")"),
+        ("fit(\"foo\")", "test.only(\"foo\")"),
+        ("it.skip(\"foo\")", "test.skip(\"foo\")"),
+        ("it.concurrent(\"foo\")", "test.concurrent(\"foo\")"),
+        ("it.only(\"foo\")", "test.only(\"foo\")"),
+        ("it.each([])(\"foo\")", "test.each([])(\"foo\")"),
+        ("it.each``(\"foo\")", "test.each``(\"foo\")"),
+        // Note: couldn't fix
+        // (
+        //     "describe.each``(\"foo\", () => { it.each``(\"bar\") })",
+        //     "describe.each``(\"foo\", () => { test.each``(\"bar\") })",
+        // ),
+        (
+            "describe.each``(\"foo\", () => { test.each``(\"bar\") })",
+            "describe.each``(\"foo\", () => { it.each``(\"bar\") })",
+        ),
+        (
+            "
+                describe.each()(\"%s\", () => {
+                    test(\"is valid, but should not be\", () => {});
+                    it(\"is not valid, but should be\", () => {});
+                });
+            ",
+            "
+                describe.each()(\"%s\", () => {
+                    it(\"is valid, but should not be\", () => {});
+                    it(\"is not valid, but should be\", () => {});
+                });
+            ",
+        ),
+        (
+            "
+                describe.only.each()(\"%s\", () => {
+                    test(\"is valid, but should not be\", () => {});
+                    it(\"is not valid, but should be\", () => {});
+                });
+            ",
+            "
+                describe.only.each()(\"%s\", () => {
+                    it(\"is valid, but should not be\", () => {});
+                    it(\"is not valid, but should be\", () => {});
+                });
+            ",
+        ),
+        // Note: couldn't fix, because the fixer couldn't be set option `fn=it`
```

Comment:
Let me try to support options in fixer, could you add TODO flag?

Reply:
Done.

Reply:
I've add a `todo` comment under this comment.